sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -79,13 +79,65 @@ enum {
 
 #define SOF_SDW_MAX_DAI_NUM		8
 
+static const struct snd_soc_dapm_widget cs_spk_widgets[] = {
+	SND_SOC_DAPM_SPK("Speakers", NULL),
+};
+
+static const struct snd_soc_dapm_widget cs42l43_dmic_widgets[] = {
+	SND_SOC_DAPM_MIC("DMIC", NULL),
+};
```

Comment:
Would it make sense to just make a generic onboard mic option and update cs42l43 to use that? Probably with a PIN_SWITCH too. Also do we actually need to differentiate between AMIC and DMIC? Maybe we should just all it "On-board Mic" or something?

Reply:
> Would it make sense to just make a generic onboard mic option and update cs42l43 to use that? Probably with a PIN_SWITCH too. Also do we actually need to differentiate between AMIC and DMIC? Maybe we should just all it "On-board Mic" or something?

renamed cs42l43_dmic_widgets to generic_dmic_widgets.